lib/Cake/Model/Datasource/Database/Sqlite.php

```diff
@@ -63,13 +63,17 @@ class Sqlite extends DboSource {
 /**
  * SQLite3 column definition
  *
+ * @link https://www.sqlite.org/datatype3.html Datatypes In SQLite Version 3
```

Comment:
Should be done by a sniffer later. Now Just unnecessary Work and waste of time IMO.